docs/failover-cluster.md

```diff
@@ -145,58 +142,62 @@ sc create <database name> start=auto binPath="<path to starservice.exe> service
 ```
 starservice.exe install <database name> <path to the database>
 ```
-   
-#### 4. Create a new cluster resource group
+
+#### 4\. Create a new cluster resource group
 
 ```
 Add-ClusterGroup Starcounter
 ```
-  
-#### 5. Create and configure the IP Adress resource
+
+#### 5\. Create and configure the IP Adress resource
 
 ```
 Add-ClusterResource -Name "IP Address" -ResourceType "IP Address" -group "Starcounter"
 Get-ClusterResource "IP Address" | Set-ClusterParameter –Multiple @{"Address"="<ip address>";"SubnetMask"="<subnet mask>";"EnableDhcp"=0}
 ```
-  
-#### 6. Create and configure the Database script resource
+
+#### 6\. Create and configure the Database script resource
 
 Copy the `scripts` folder from the previously downloaded archive to all nodes. The local path should be the same on all nodes. Don't use the CSV volume as it will complicate resource upgrade and troubleshooting.
 
 **Then**:
-        
+
 ```
 Add-ClusterResource -name Database -group "Starcounter" -ResourceType "Generic Script"
 Get-ClusterResource "Database" | Set-ClusterParameter  -name ScriptFilepath -value "<path to script.js>"
 Get-ClusterResource "Database" | Set-ClusterParameter  -name DbName -value "<database name>" -create
 ```
 
-#### 7. Create and configure the `WebApp` resource
+#### 7\. Create and configure the `WebApp` resource
 
 Copy the `WebApp` files to all nodes. The local path should be the same across nodes. Don't use CSV volume as it will complicate resource upgrade and troubleshooting.
 
 **Then**:
-   
+
 ```
 Add-ClusterResource -Name "WebApp" -Group "Starcounter" -ResourceType "Generic Application"
 Get-ClusterResource "WebApp" | Set-ClusterParameter -Name CommandLine -Value "<path to webapp.exe>"
 ```
-  
-#### 8. Setup resource dependencies
+
+#### 8\. Setup resource dependencies
 
 ```
 Set-ClusterResourceDependency -Resource WebApp -Dependency "([IP Address]) AND ([Database])"
 ```
-  
-#### 9. Start the group
+
+#### 9\. Start the group
 
 ```
 Start-ClusterGroup Starcounter
 ```
 
-#### 10. Extra notes and resources
+#### 10\. Extra notes and resources
 
 - Make sure to specify a reasonable configuration for the maximum allowed failures over time for the required resources. The default configuration is very limiting.
 - Make sure to have at least three nodes in a cluster, or a file share witness to keep the cluster alive when a node goes down.
 - [Deploying Scale-Out file Server in Cluster (Server 2016)](https://geekdudes.wordpress.com/2016/12/26/deploying-scale-out-file-server-in-cluster-server-2016/).
-- [How to Configure a File Share Witness for a Hyper-V Cluster](https://www.altaro.com/hyper-v/file-share-witness-hyper-v-cluster/).
\ No newline at end of file
+- [How to Configure a File Share Witness for a Hyper-V Cluster](https://www.altaro.com/hyper-v/file-share-witness-hyper-v-cluster/).
+
+## Starcounter failover on Linux
+
+Starcounter 3 Release Candidate does not yet support failover on Linux operating systems out of the box.
```

Comment:
It would be nice to also add something like:

>Please contact us if you have a production Linux environment which requires failover.

But it's up to you to add it or not.

Reply:
done